components/AboutMe.vue

```diff
@@ -3,17 +3,18 @@
     <IconBase class="text-sunset w-full h-16 -translate-y-4">
       <IconArrow />
     </IconBase>
-    <SectionContent class="flex flex-col lg:flex-row items-center">
-      <figure class="relative w-2/3 sm:w-1/2 mb-12 pr-8 lg:pr-16">
-        <img
-          srcset="~/assets/images/mobile/ironleaves-photography-ueber-mich-tamara-loeffen-SW-mobile.jpg 768w,
-            ~/assets/images/desktop/ironleaves-photography-ueber-mich-tamara-loeffen-SW.jpg 1920w"
-          sizes="(max-width: 768px) 768px,
-            1920px"
-          alt="Tamara Loeffen Portrait">
+    <SectionContent class="slide-up flex flex-col lg:flex-row items-center md:gap-y-8">
+      <figure class="relative w-2/3 sm:w-1/2 mb-12 pr-8 lg:pr-16 slide-up" data-delay="0.3">
```

Comment:
brauchts den delay? bild ist ja niedriger

```suggestion
      <figure class="slide-up relative w-2/3 sm:w-1/2 mb-12 pr-8 lg:pr-16" data-delay="0.3">
```